### PR TITLE
Add AsyncQuidax class

### LINKED ISSUE #9: Implement AsyncQuidax wrapper

### PR DESCRIPTION
**Pull request by beekaz**
This resolves #9

### CODE REVIEW

File: pyquidax/quidax.py [hunk 1]
```diff
@@ -62,3 +62,36 @@ def withdrawal_fee(self, currency: Currency):
 
 class AsyncQuidax:
```
Comment (gray-adeyi) [inline pyquidax/quidax.py:63]: Blessing it's not just about coping and pasting. I want you to understand why we do what we do. This issue is about implementing AsyncQuidax you've already seen an empty class defined so why create a new one with the same name

File: pyquidax/quidax.py [hunk 2]
```diff
@@ -62,3 +62,36 @@ def withdrawal_fee(self, currency: Currency):
...
+            method=HTTPMethod.GET,
+        )
+
+    async def withdrawal_fee(self, currency: Currency):
+        return await self._api_call(
+            url=f"{self.base_url}/fee?currency={currency}", method=HTTPMethod.GET
+        )
+
+
+class AsyncQuidax:
```
Comment (gray-adeyi) [inline pyquidax/quidax.py:96]: This too. Do we really need it?

**Review by gray-adeyi — CHANGES_REQUESTED** [review]

**Comment by gray-adeyi** [comment]
Click on the resolve conversation button for the requested changes you've made

**Comment by gray-adeyi** [comment]
I wrote review comments for you to make some corrections. for the ones you've fixed click on the `resolve conversation` button

**Review by beekaz — COMMENTED** [review]
resolves #9

**Comment by gray-adeyi** [comment]
Nicely done 👍🏿

**Pull request merged**